src/lib/getVars.ts

```diff
@@ -28,10 +35,13 @@ export const getVars = (): Vars => {
   const options = {
     key: core.getInput('key') || 'no-key',
     path: core.getInput('path'),
+    strategy: core.getInput('strategy') as Strategy,
   }
 
   if (!options.path) {
     throw new TypeError('path is required but was not provided.')
+  } else if (!Object.values(Strategy).includes(options.strategy)) {
+    throw new TypeError(`Unknown strategy ${options.strategy}`)
   }
```

Comment:
These are unrelated code-paths, let's separate to avoid bugs in the future.

```suggestion
  if (!options.path) {
    throw new TypeError('path is required but was not provided.')
  }
  
  if (!Object.values(Strategy).includes(options.strategy)) {
    throw new TypeError(`Unknown strategy ${options.strategy}`)
  }
```